Add mission control package documentation (#128)
ChrisScianna/ROS-Underwater-RnD: mission_control/README.md

# Mission Control
The mission control package provides an implementation of missions using behavior trees
It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) library.  

## Features

> - The mission control implements the interface with the Jaus Ros Bridge ROS Package.

Reviewer: @patapufate how about: 
```suggestion
- Topic interface for mission loading, execution, abortion, and monitoring, compliant with that of its predecessor (`mission_manager`)
```

Author: Fixed

> The behaviors/actions implemented were:

Reviewer: @patapufate how about 
```suggestion
- Actions supported are:
```
and then indent the list below.

Author: Done

- Abort
- Attitude Servo
- Fix Rudder
- Go To Waypoint
- Payload Command
- Set Altitude Heading

> - Set Depth Heading

Reviewer: @patapufate a one liner explaining what the action does would be great.

Author: Fixed

> - The missions can be designed using Groot

Reviewer: @patapufate I think one bullet should talk about missions being described as behavior trees in XML.

Author: Done

> 

Reviewer: @patapufate meta: I'd also mention it's fail safe :)

Author: Done

## Features

- Topic interface for mission loading, execution, abortion, and monitoring, compliant with that of its predecessor (`mission_manager`)

- Actions supported are:
&nbsp; - **Abort**. Set the fins to surface and Thruster Velocity to 0 RPM.  
&nbsp; - **Attitude Servo**. It sends the command to actuator setting the roll, pitch, yaw and speed parameters.  
&nbsp; - **Fix Rudder**. It sends the command to actuators setting the rudder, depth and speed.  
&nbsp; - **Go To Waypoint**. It sends command to autopilot to go to altitude, latitude, longitude and speed.  
&nbsp; - **Payload Command**. It sends a string command to Payload  
&nbsp; - **Set Altitude Heading**. It sends command to actuator setting the altitude, heading and speed.  
&nbsp; - **Set Depth Heading**. It sends command to actuator setting the depth, heading and speed.  

- The missions can be designed using Groot

> - The mission are described as behavior trees in XML

Reviewer: @patapufate nit: 
```suggestion
-Mission are described as behavior trees in XML.
```

Author: Fixed

> - Abort in the event of unexpected anomalies the vehicle will undertake an abort behaviour going to surface

Reviewer: @patapufate nit:
```suggestion
- Abort missions and go to surface in the event of unexpected anomalies (e.g. system faults)
```

Author: Fixed
